lib/galaxy/tool_util/verify/interactor.py

```diff
@@ -900,20 +902,34 @@ def verify_tool(tool_id,
         "test_index": test_index,
     }
     client_config = client_test_config.get_test_config(job_data)
+    skip_message = None
     if client_config is not None:
         job_data.update(client_config)
         skip_message = job_data.get("skip")
-        if skip_message:
-            job_data["status"] = "skip"
-            register_job_data(job_data)
-            return
+
+    if not skip_message and skip_with_reference_data:
+        required_data_tables = tool_test_dict.get("required_data_tables")
+        required_loc_files = tool_test_dict.get("required_loc_files")
+        # TODO: actually hit the API and see if these tables are available.
```

Comment:
```suggestion
```
Even if they are available it's unlikely that they would contain the minimal examples required by tool tests (hg19, sacCer2 etc. are not uncommon unfortunately).

Reply:
That is insane that we do that right? How is that acceptable 😅